src/pages/linux/install-and-configure-apache-server-in-linux/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Install and Configure Apache Server in Linux
+---
+
+<div class="para">Apache HTTP Server, colloquially called Apache, is free and open-source 
+cross-platform web server software.
+Apache is developed and maintained by an 
+open community of developers under the auspices of the Apache Software Foundation.</div>
+
+# Install and Configure Apache Server in Linux
```

Comment:
~# Install and Configure Apache Server in Linux~

`## Install and Configure Apache Server in Linux `